### ISSUE TITLE
Update to Haxe 4.1.1 causes build to HTML5 error

### ISSUE THREAD

**Issue by themelz** [openfl/openfl#2354]
Just upgraded to Haxe 4.1.1 and subsequent build gave this error:

> haxe/lib/openfl/8,9,6/src/openfl/display/Stage.hx:1426: characters 46-59 : haxe.CallStack has no field lastException

Also tried Haxe 4.1.0, got same problem. Previous version that worked was 4.0.5.

Solutions beside reverting to 4.0.5?

**Comment by Snky** [comment]
How can I confirm my Haxe version? (CMD->haxelib reports Haxe Library Manager 4.0.2)
I believe I recently patched my portable Haxe to 4.1.1 (just checked CHANGES.txt)
I do not have any issues currently with running an application using OpenFL.

What I did additionally that may help is, I downloaded the latest zips for all in my lib folder, even if a new version was not officially released to be downloaded through 'haxelib upgrade'.
